### DISCUSSION TITLE
Change background with Imagick

### DISCUSSION THREAD

**Discussion by Stajor** [chillerlan/php-qrcode#281] [Q&A] [answered]
Hi, I need help from the community.
I'm trying to change the background without success. 
I tried different variations, read questions here on the topic.

Version: chillerlan/php-qrcode: 5.0.2
Code:

`
$options = new QROptions();
$options->outputType = QROutputInterface::IMAGICK;
$options->outputInterface = QRImagick::class;
$options->quality = 100;
$options->addQuietzone = false;
$options->imageTransparent = false;
$options->transparencyColor = "red";
$options->bgColor = "red";
$options->scale = 100;

return (new QRCode($options))->render($url);
`

**Comment by codemasher** [comment, marked as answer]
That's because your QR Code has no visible background, it's covered by black and white modules. Try setting `drawLightModules` to `false` and you'll see the background color.

**Reply by Stajor** [reply]
Thank you very much, it works.
